docs/getting-started/create-an-app.md

```diff
@@ -0,0 +1,81 @@
+# Backstage App
+
+If you want to get set up quickly with your own Backstage project you can create a Backstage App.
+
+A Backstage App is a monorepo setup with `lerna` that includes everything you need to run Backstage in your own environment.
+
+## Create an app
+
+If you have [`npx`](https://github.com/npm/npx) installed simply run:
```

Comment:
npx comes bundled with npm/node, so no need to link to it. Could maybe reiterate that node v12 is needed